app/src/androidTest/java/ch/epfl/sweng/SDP/game/WaitingPageActivityTest.java

```diff
@@ -215,13 +227,13 @@ public void testOnState1Change() {
     }
 
     @Test
-    public void testOnState2Change() {
```

Comment:
Where is `testOnStateChange2()` now ?

Reply:
I changed the name of this function to testOnState3Change because the test was failing when it was called testOnState2Change

Reply:
this is so weird, what would it fail because of 2 or 3 in the name of the method?

Reply:
I believe it has something to do with my IDE. It's possible that Android Studio cached the old version compiled version and did not update when I changed the code

Reply:
Try to change it to `testOnState2Change()` it should work

Reply:
Alright I changed this